build.number

```diff
@@ -5,5 +5,5 @@ version.patch=0
 # This is the -N part of a version.  if it's 0, it's dropped from maven versions.
 version.bnum=0
 
-# Note: To build a release run ant with -Dbuild.release=true
-# To build an RC, run ant with -Dmaven.version.suffix=-RCN
+# To build a release, see scripts/jobs/scala-release-2.11.x-build
```

Comment:
Purely FYI, this line is also obsolete in 2.11.x -- I removed version numbers from script names for this reason, but forgot to update the comment.